package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watch",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
This was correct. v0.4.1 doesn't exist yet.

Reply:
Wouldn't this fix cause a new release?

Reply:
No, there's a separate release process I'll run. It updates this, the changelog, creates a release commit and git tag.

Reply:
Thank you!